src/index.ts

```diff
@@ -203,6 +203,135 @@ export class Datastore {
         await this._contract.setFilename(fileId, newName)
     }
 
+    /**
+     * Creates a new group of entities
+     * @param groupName Name of the group
+     */
+    async createGroup(groupName: string) {
+        await this._initialize()
+
+        await this._contract.createGroup(groupName)
+    }
+
+    /**
+     * Deletes a group
+     * @param groupId Id of the group
+     */
+    async deleteGroup(groupId: number) {
+        await this._initialize()
+
+        await this._contract.deleteGroup(groupId)
+    }
+
+    /**
+     * Rename an existing group
+     * @param groupId Id of the group to rename
+     * @param newGroupName New group name
+     */
+    async renameGroup(groupId: number, newGroupName: string) {
+        await this._initialize()
+
+        await this._contract.renameGroup(groupId, newGroupName)
+    }
+
+    /**
+     * Returns an array of all the groups infos
+     */
+    async getGroupsInfos() {
+        await this._initialize()
+
+        let groups = []
+        let groupsIds = await this._contract.getGroups()
+        for(var i = 0; i < groupsIds.length; i++) {
+            let getGroup = await this._contract.getGroup(groupsIds[i])
+            let group = {
+                id: groupsIds[i],
+                name: getGroup[0],
+                entities: getGroup[1]
+            }
+            if(group.name !== 0)
+                groups.push(group)
+        }
+        return groups
+    }
+
+    /**
+     * Returns the entities from a group
+     * @param groupId Id of the group to get entities from
+     */
+    async getGroup(groupId: number) {
+        await this._initialize()
+
+        await this._contract.getGroup(groupId)
+    }
+
+    /**
+     * Returns an entity from the `entities` array 
+     * @param groupId Id of the group to get entity from
+     * @param entityIndex Index of that entity in the `entities` array
+     */
+    async getGroupEntity(groupId: number, entityIndex: number) {
+        await this._initialize()
+
+        await this._contract.getGroupEntity(groupId, entityIndex)
+    }
+
+    /**
+     * Returns the number of entities in the specified group
+     * @param groupId Id of the group to get count from
+     */
+    async getGroupCount(groupId: number) {
```

Comment:
Same thing here, don't think this function is necessary. It really is an internal implementation helper